embc-app/ClientApp/src/app/volunteer/pages/referral-view/referral-view.component.ts

```diff
@@ -13,21 +15,25 @@ import {
   templateUrl: './referral-view.component.html',
   styleUrls: ['./referral-view.component.scss']
 })
-export class ReferralViewComponent implements OnInit {
+export class ReferralViewComponent implements OnInit, OnDestroy {
 
-  path: string = null; // for relative routing
+  private confirmModal: NgbModalRef = null;
+  private path: string = null; // for relative routing
   registrationId: string = null;
   referralId: string = null;
   referral: Referral = null;
   loading = true;
   deactivating = false;
+  reason = 'null';
 
   constructor(
     private route: ActivatedRoute,
     private router: Router,
+    private modals: NgbModal,
     private authService: AuthService,
     private uniqueKeyService: UniqueKeyService,
     private referralService: ReferralService,
+    private notifications: NotificationQueueService,
```

Comment:
We should aim for consistent service variable names. Angular's official suggestion for naming is like this:
    `private notificationQueueService: NotificationQueueService,`
I tend to agree with them because then we can see in functions what is or is not a service and have a consistent way to search through the project for references to a service. Otherwise you have to go find the service in the constructor and search on the variable that is beside it.

Reply:
Sure. As discussed, we won't hold back this PR for this but we will update in future updates.